examples/handlebars/docs/01-index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Fractal default example
+---
+
+This is your index page. You can edit its contents at `docs/01-index.hbs`
```

Comment:
the extension should be md here i think? is this a bug in our `new` command?

Reply:
Yes, was a bug, fixed it.